chispa/number_helpers.py

```diff
@@ -0,0 +1,12 @@
+import math
+
+
+def isnan(x):
+    try:
+        return math.isnan(x)
+    except:
```

Comment:
A naked `except` is generally undesirable because it'll catch absolutely everything, including things like `KeyboardInterrupt`. You probably want `except ValueError`.

Reply:
Fixed, good catch